Extensions/xkit_main.js

```diff
@@ -166,7 +166,7 @@ XKit.extensions.xkit_main = new Object({
 		}
 
 		try {
-			eval(xkit_main.script);
+			eval(xkit_main.script + "\n//# sourceURL=xkit/"+extension_id+".js");
```

Comment:
Some of these don't actually happen, but I added console.log(xkit_main.script) to all of them and the only one of them that is running definitely has xkit main's script that's being evaluated,